src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -100,10 +100,14 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
             this, SLOT(slotSetRateRange(int)));
 
     // Set default range as stored in config file
-    if (m_pConfig->getValueString(ConfigKey("[Controls]", "RateRangePercent")).length() == 0) {
+    if (!m_pConfig->exists(ConfigKey("[Controls]","RateRangePercent")) ||
+        m_pConfig->getValueString(ConfigKey("[Controls]", "RateRangePercent")).length() == 0) {
         // Fall back to old [Controls]RateRange
-        if (m_pConfig->getValueString(ConfigKey("[Controls]", "RateRange")).length() == 0) {
-            m_pConfig->set(ConfigKey("[Controls]", "RateRangePercent"), ConfigValue(8));
+        if (!m_pConfig->exists(ConfigKey("[Controls]","RateRange")) ||
+            m_pConfig->getValueString(ConfigKey("[Controls]", "RateRange")).length() == 0) {
+            int rateRangePercent = 8;
+            m_pConfig->set(ConfigKey("[Controls]", "RateRangePercent"), ConfigValue(rateRangePercent));
+            slotSetRateRangePercent(rateRangePercent);
```

Comment:
Nope!
As I reported, it affects other settings as well, i.e position display. Don't know how to approach that.